services/QuillLMS/app/workers/snapshots/cache_snapshot_count_worker.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+
+module Snapshots
+  class CacheSnapshotCountWorker
+    include Sidekiq::Worker
+
+    DEFAULT_CACHE_EXPIRATION = 24.hours
+    PUSHER_EVENT = 'admin-snapshot-count-cached'
+
+    QUERIES = {
```

Comment:
Do you have a good sense of how wee build a list of "human readable slugs" that refer to each report?  We need something for the front-end to be able to pass to say "I want this specific report" since we don't just automatically run all the queries on any single trigger at the moment.

Reply:
One option is to use `<nice_snake_case>.camelize.constantize` in Rails, so you can send `<nice_snake_case>` from the front end